null is not nullable

Using checker 3.12.0. I would expect `null` to be nullable, however this errors with

```
/home/jpschewe/projects/checker-bugs/src/main/java/net/mtu/eggplant/checker/NullNotNullable.java:9: error: [argument.type.incompatible] incompatible argument for parameter variableScope of NullNotNullable.
    this(null);
         ^
  found   : null (NullType)
  required: @UnderInitialization @Nullable Object
```


```java
import org.checkerframework.checker.initialization.qual.UnderInitialization;
import org.checkerframework.checker.nullness.qual.Nullable;

public class NullNotNullable {

  public NullNotNullable() {
    this(null);
  }

  protected NullNotNullable(final @UnderInitialization @Nullable Object variableScope) {
  }
  
}
```

Thanks for reporting this!  This is a bug, but the problem is with the Initialization Checker.  Here's the error when using `-AprintAllQualifiers`:
```
NullNotNullable.java:7: error: [argument.type.incompatible] incompatible argument for parameter variableScope of NullNotNullable.
    this(null);
         ^
  found   : @Initialized @Nullable NullType
  required: @UnderInitialization @Nullable Object
1 error
```

--- a/checker-qual/src/main/java/org/checkerframework/checker/initialization/qual/FBCBottom.java
+++ b/checker-qual/src/main/java/org/checkerframework/checker/initialization/qual/FBCBottom.java
@@ -5,6 +5,8 @@ import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.LiteralKind;
+import org.checkerframework.framework.qual.QualifierForLiterals;
 import org.checkerframework.framework.qual.SubtypeOf;
 import org.checkerframework.framework.qual.TargetLocations;
 import org.checkerframework.framework.qual.TypeUseLocation;
@@ -23,4 +25,5 @@ import org.checkerframework.framework.qual.TypeUseLocation;
 @Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
 @TargetLocations({TypeUseLocation.EXPLICIT_LOWER_BOUND, TypeUseLocation.EXPLICIT_UPPER_BOUND})
 @SubtypeOf({UnderInitialization.class, Initialized.class})
+@QualifierForLiterals(LiteralKind.NULL)
 public @interface FBCBottom {}

--- a/checker/tests/nullness/init/Issue4567.java
+++ b/checker/tests/nullness/init/Issue4567.java
@@ -1,0 +1,11 @@
+import org.checkerframework.checker.initialization.qual.UnderInitialization;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class Issue4567 {
+
+  public Issue4567() {
+    this(null);
+  }
+
+  protected Issue4567(final @UnderInitialization @Nullable Object variableScope) {}
+}

